addons/digest/tests/test_digest_kpi.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.digest.tests.common import TestDigestCommon
+
+
+class TestDigest(TestDigestCommon):
+
+    def test_kpi_mail_message_total_value(self):
+        # Sanity check
+        initial_values = self.all_digests.mapped('kpi_mail_message_total_value')
```

Comment:
That probably highly depends on installed modules :D would reset messages in setupclass (either count existing one and check new_count = old_count + create_cnt, either unlink all existing one)